### ISSUE TITLE
Rework target architecture

### ISSUE THREAD

**Issue by arithx** [coreos/mantle#1069]
# Feature Request #

The target architecture in mantle is currently specified via the `board` flag and has `-usr` appended to the end of the architecture. This was due to compatability constraints in Container Linux that we are no longer beholden to after branching.

## Desired Feature ##

 - Change the parameter name
 - Drop `-usr` suffixes

**Comment by Prashanth684** [comment]
@arithx Just trying to understand..does this mean the master branch will not support CL anymore and is purely for FCOS and RCOS?

**Comment by arithx** [comment]
Yes, `master` is no longer guaranteed to be stable for Container Linux, we will still be backporting (and testing for CL) platform / command related changes that are needed to the `cl` branch.

**Comment by Prashanth684** [comment]
I started working on this rework and had a few questions:

-  is there a comprehensive test suite that I can run my local changes on ? 
- There are a few URLs which reference the `-usr` board names. Are they applicable only for CL?

**Comment by arithx** [comment]
> is there a comprehensive test suite that I can run my local changes on ?

Unfortunately no; there's a few unit tests but overall I'd suggest just running `kola` (particularly on `qemu*` platforms).

> There are a few URLs which reference the -usr board names. Are they applicable only for CL?

Yes, `-usr` was a suffix we had for architecture names in CL and that's one of the things we want to drop as a result of this rework.
